src/darwin/mod.rs

```diff
@@ -334,18 +271,41 @@ pub fn descriptors_for_family(family: &str) -> Vec<Descriptor> {
     out
 }
 
+// The AppleFontSmoothing user default controls font smoothing on macOS. By default it is unset,
+// and the system behaves as though it is set to 2, which means a medium level of font smoothing.
+// We will interpret anything other than 0 as enabling font smoothing, and 0 as disabling it. Note
+// that setting this default as a false boolean value *does not* disable font smoothing; it must be
+// an integer 0.
+static FONT_SMOOTHING_ENABLED: Lazy<bool> = Lazy::new(|| unsafe {
+    let font_smoothing_key = NSString::alloc(nil).init_str("AppleFontSmoothing");
+    let font_smoothing_value: id =
+        msg_send![id::standardUserDefaults(), objectForKey: font_smoothing_key];
+    if msg_send![font_smoothing_value, isKindOfClass: class!(NSNumber)] {
+        match CStr::from_ptr(msg_send![font_smoothing_value, objCType]).to_bytes() {
+            b"q" | b"l" | b"i" | b"s" => {
```

Comment:
Could you add description wrt those magic strings?

Reply:
Done.